core/src/main/scala/org/apache/spark/deploy/worker/ExecutorRunner.scala

```diff
@@ -143,11 +146,15 @@ private[deploy] class ExecutorRunner(
    */
   private def fetchAndRunExecutor() {
     try {
+      val resourceFile = ResourceUtils.prepareResourceFile(
+        SPARK_EXECUTOR_PREFIX, resources, executorDir)
       // Launch the process
+      val arguments = appDesc.command.arguments ++
+        Seq("--resourcesFile", resourceFile.getAbsolutePath)
```

Comment:
similar to above, if no resources better just to skip sending this and not creating a file

Reply:
Good idea.